lib/core/src/model.rs

```diff
@@ -194,31 +194,42 @@ pub struct ConnectRequest {
     pub config: Config,
 }
 
+/// The receive methods supported by the SDK
```

Comment:
```suggestion
/// The send / receive methods supported by the SDK
```

This will probably be used for `Payment`, so it covers the send methods supported as well.